src/chapters/azure-intro/studio.rst

```diff
@@ -189,7 +63,7 @@ Clone your partner's repo. This time clone it into a different directory by pref
 
 Navigate to your partner's cloned repo directory from the command line. Once in the directory run the publish command to package the API into its executable. After publishing execute the published executable just as you did earlier.
 
-As the final step open the Swagger documentation page `https://localhost:5001`_ in your browser.
+As the final step open the Swagger documentation page `https://localhost:5001 <https://localhost:5001>`_ in your browser.
```

Comment:
this link is broken. idk how to direct link properly

Reply:
this link is not broken for me and the RST sytanx is correct.